_base/.env.example

```diff
@@ -30,3 +30,4 @@ NEXT_PUBLIC_DEMO_MODE="false"
 ### Database configuration
 
 DATABASE_URL="postgresql://postgres@localhost/issuing_treasury"
+POSTGRES_URL=${DATABASE_URL}
```

Comment:
essentially renames this env var to `POSTGRES_URL` and aliases `DATABASE_URL` to it -- because that's what Vercel calls it by default. it's configurable, but less friction for the user deploying to just leave it as default